doc/source/operating.rst

```diff
@@ -153,7 +153,7 @@ For example, if you want to keep a year of data with a one minute resolution::
 
 Then::
 
-    size in bytes = 525 600 bytes × 6 = 3 159 600 bytes = 3 085 KiB
+    size in bytes = 525 600 bytes × 8 = 4 204 800 bytes = 4 106 KiB
```

Comment:
Ok, so the code is wrong but the fix too. I thought it was about the number of archive policies, but it's indeed a number of bytes, but the bytes are on the wrong side. It should be:

```size in bytes = 525 600 points × 8 bytes = 4 204 800 bytes = 4 106 KiB```